src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -136,6 +137,16 @@ def go_from_trigger_mozilla_central(self):
         r.raise_for_status()
         push_id = r.json()['pushid']
 
+        for artifact in self.artifactsHandler.get():
+            if 'jsvm' in artifact:
+                with zipfile.ZipFile(artifact, 'r') as zf:
+                    for file_name in zf.namelist():
+                        with zf.open(file_name, 'r') as fl:
+                            source_files = [line[3:-1].decode('utf-8') for line in fl if line.startswith(b'SF:')]
```

Comment:
You can use `rstrip` here to be sure you are only removing the \r and/or \n characters, instead of manually forcing the removal of the last character.